Files/UserControls/Widgets/LibraryCards.xaml

```diff
@@ -108,7 +108,7 @@
                                         <MenuFlyoutItem
                                             x:Name="OpenInNewPane"
                                             x:Uid="SideBarOpenInNewPane"
-                                            x:Load="{x:Bind HasPath, Mode=OneWay}"
```

Comment:
The default `Mode` is `OneTime` so if the value changes, the UI won't be updated. (And in other places too)

Reply:
Here it's redundant because I checked and the ViewModel doesn't implement INotifyPropertyChanged and the property isn't a DependencyProperty so it wouldn't work.